content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+
+    {{< text bash >}}
+    $ export WORK_DIR=$(pwd)
+    $ CLUSTER_NAME=$(kubectl config view --minify=true -o "jsonpath={.clusters[].name}")
+    $ CLUSTER_NAME="${CLUSTER_NAME##*_}"
+    $ export KUBECFG_FILE=${WORK_DIR}/${CLUSTER_NAME}
+    $ SERVER=$(kubectl config view --minify=true -o "jsonpath={.clusters[].cluster.server}")
+    $ NAMESPACE=istio-system
+    $ SERVICE_ACCOUNT=istio-multi
+    $ SECRET_NAME=$(kubectl get sa ${SERVICE_ACCOUNT} -n ${NAMESPACE} -o jsonpath='{.secrets[].name}')
+    $ CA_DATA=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['ca\.crt']}")
+    $ TOKEN=$(kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o "jsonpath={.data['token']}" | base64 --decode)
+    {{< /text >}}
+
+    __NOTE__: An alternative to `base64 --decode` is `openssl enc -d -base64 -A` on many systems.
+
+1.  Create a `kubeconfig` file in the working directory for the service account `istio-multi`:
+
+    {{< text bash >}}
+    $ cat <<EOF > ${KUBECFG_FILE}
+    apiVersion: v1
+    clusters:
+       - cluster:
+           certificate-authority-data: ${CA_DATA}
+           server: ${SERVER}
+         name: ${CLUSTER_NAME}
+    contexts:
+       - context:
+           cluster: ${CLUSTER_NAME}
+           user: ${CLUSTER_NAME}
+         name: ${CLUSTER_NAME}
+    current-context: ${CLUSTER_NAME}
+    kind: Config
+    preferences: {}
+    users:
+       - name: ${CLUSTER_NAME}
+         user:
+           token: ${TOKEN}
+    EOF
+    {{< /text >}}
+
+At this point, the remote clusters' `kubeconfig` files have been created in the `${WORK_DIR}` directory.
+The filename for a cluster is the same as the original `kubeconfig` cluster name.
+
+## Configure Istio control plane to discover the remote cluster
+
+Create a secret and label it properly for each remote cluster:
+
+{{< text bash >}}
+$ kubectl create secret generic ${CLUSTER_NAME} --from-file ${KUBECFG_FILE} -n ${NAMESPACE}
+$ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
+{{< /text >}}
+
+## Deploy Bookinfo Example Across Clusters
+
+1.  Install bookinfo on the first cluster.  Remove the `reviews-v3` deployment to deploy on remote:
+
+    {{< text bash >}}
+    $ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+    $ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+    $ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+    $ kubectl delete deployment reviews-v3
+    {{< /text >}}
+
+1.  Install the `reviews-v3` deployment on the remote.  _Note:_ The `ratings` service definition is added
+    to the remote cluster because `reviews-v3` is a client of `ratings` and creating the service object
+    creates a DNS entry.  The Istio sidecar in the `reviews-v3` pod will determine the proper `ratings`
+    endpoint after the DNS lookup is resolved to a service address:
+
+    {{< text bash >}}
+    $ cat <<EOF > $HOME/reviews-v3.yaml
```

Comment:
Readers can create the file in the remotes whichever way they want, having them do it with a cat command can be confusing or annoying for some.

Reply:
covered in prior comment about copy/paste.  Any others have opinions on which way to go with yaml blocks like this?

Reply:
We use #cat throughout the site at this point. There are alternative however as decribed in the following two links:

https://preliminary.istio.io/about/contribute/writing-a-new-topic/#embedding-preformatted-blocks
https://preliminary.istio.io/about/contribute/writing-a-new-topic/#linking-to-other-pages

Reply:
In this case, you could actually just list a chunk of YAML, and the user can click on the download button to download the file. You can use the downloadas argument in the markup to control the file name used by the browser during the download.

The chunk of YAML can be put inline, or can be ingested from someplace else. The better choice is to have some test code somewhere using the YAML to ensure it is correct on an on-going basis, and then pull in that YAML file to the doc site. In  lieu of that, just a block of YAML in the markdown will work fine.

Reply:
@geeknoid -- thanks!  Great info!  Yeah, I originally followed the cat approach as in the other content.  But the yaml format text block that can be downloaded as a file per that link is probably better.  I've separated to different bullets per @rcaballeromx and made the downloadable yaml block.  It wouldn't merge in time for 1.0 to get anything in istio/istio samples/bookinfo/ so in doc only is best for now, IMO.